sdk/identity/azure-identity-extensions/README.md

```diff
@@ -25,6 +25,10 @@ Azure Identity Extensions contains a common template framework for users to get
 use the token as a password. For example, to connect Azure hosted MySQL, get a token from Azure AD and use the
 token as a password to connect with MySQL.
 
+## Architecture
+This picture shows how the Azure Identity Extensions (JDBC authentication plugin) authenticates with MSI.
```

Comment:
```suggestion
This picture shows how the Azure Identity Extensions (JDBC authentication plugin) authenticates with managed identity.
```

Reply:
Since the `Azure Identity Extensions` will cover more scenarios/services than JDBC plugin, so IMO the `Azure Identity Extensions (JDBC authentication plugin)` is not or will not be accurate.

Reply:
Yes, but the picture only shows jdbc authentication.

Reply:
Maybe we should rephrase this `Azure Identity Extensions (JDBC authentication plugin)`. The `()` will make people think these two are equal.

Reply:
how about `Azure Identity Extensions (it contains JDBC authentication plugin)`

Reply:
How about adding a h3 under the `Architecture` called `Authenticating with JDBC`, and changing the sentence to `This picture shows how the JDBC authentication plugins provided by Azure Identity Extensions authenticate with managed identity`?